docker-compose-build.yml

```diff
@@ -0,0 +1,10 @@
+version: '3.5'
+
+services:
+
+  nhsd.buyingcatalogue.ordering.api:
+    image: ${REGISTRY:-nhsd}/buying-catalogue/ordering-isapi:${TAG:-latest}
```

Comment:
would you please consider changing the image name from:
 `${REGISTRY:-nhsd}/buying-catalogue/ordering-isapi:${TAG:-latest}`
to 
`${REGISTRY:-nhsd}/buying-catalogue/ordering-api:${TAG:-latest}`

Isapi is an abbreviation we've been using for 'Identity Server API' and might cause confusion.